packages/sw-precaching/src/lib/controllers/base-cache-manager.js

```diff
@@ -13,7 +13,8 @@ class BaseCacheManager {
    */
   constructor(cacheName) {
     this._entriesToCache = new Map();
-    this._cacheName = cacheName;
+    this._requestWrapper = new RequestWrapper({cacheName});
```

Comment:
Definitely not a stupid question. It's not the default for the underlying `fetch()`, so matching `fetch()` behavior by default unless otherwise required seemed like the least-surprising thing to do.

We ran into a scenario with IOWA in which we specifically needed this set, which is how it ended up in `sw-precache` initially.

I'm open to setting different values by default in `RequestWrapper` directly if you'd like to open an issue to discuss.